data/rail_network/rail_interchange_impedances.yaml

```diff
@@ -0,0 +1,29 @@
+# Data/rules for assigning impedances at rail interchanges
+
+# Defaults for interchanges not named in the rule 260 section
+class1_to_class1_default_impedance: 1000
+class1_to_shortline_default_impedance: 350
+
+# Defaults by Rule 260 interchange
+impedance_by_rule_260_interchange: {
+    # for all nodes that
+    class1_to_class1_impedance: 250,
+    not_class1_to_class1_impedance: 275,
+    radius_to_search_miles: 20,
+
+    # Rules:  need either the lat_long or the franodeids to specify where the
+    #         the interchange occurs
+    #         the jr260 name is optional
+    #         the overrides is optional, and is a pair of rr's and impedance
+    #         Interchange links will be checked to see if the override RR pair is valid
+    interchange_rules:
+      [
```

Comment:
I appreciate this is valid yaml but we can avoid it an make it much more readable following the guidelines above.